code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -244,6 +244,20 @@
 		"<span class='userdanger'>You feel a powerful shock coursing through your body!</span>", \
 		"<span class='italics'>You hear a heavy electrical crack.</span>" \
 		)
+	if(iscarbon(pulling) && !illusion && source != pulling)

```

Comment:
source can never be pulling if it's a carbon since source is typed as an object

Reply:
the correct fix is to retype source me thinks

Reply:
They are both a reference to the same memory adress, I assume there is some implicit type casting involved at compiler level. Either way it works as expected.
Do you want me to explicitly cast these two vars to an object?

Reply:
I want source to not be typed as an object (L226) if source can be things other than an object.

Reply:
Right, yeah that makes sense. Source is only ever used for chat output anyway, before now.